### DISCUSSION TITLE
UI update

### DISCUSSION THREAD

**Discussion by masterodi** [seerge/g-helper#2526] [General]
@seerge hello,

I was thinking to help in updating the UI of the app a little bit. 

Are you open to UI changes or you prefer to leave it as is? If the answer is yes, I will come back with some mockups of the updated UI before going any further.

**Comment by Ran7IX** [comment]
yeah i think reorganising the app would be good, i suggest making menus on the left side ex; Performance, Profiles, screen and updates
